<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorWindow</class>
 <widget class="QMainWindow" name="CalculatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>613</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>450</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>705</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #b2a7dd;
border-radius: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Open Sans</family>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #6656be;
color: #dddbe9;
border-radius: 10px;</string>
            </property>
            <property name="text">
             <string>Saldo Anda: Rp. 100.000.000</string>
            </property>
            <property name="margin">
             <number>5</number>
            </property>
            <property name="indent">
             <number>15</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="">
            <property name="styleSheet">
             <string notr="true">background-color: #6656be;</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Open Sans</family>
                 <pointsize>15</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #dddbe9;</string>
               </property>
               <property name="text">
                <string>Nama Barang:</string>
               </property>
               <property name="margin">
                <number>5</number>
               </property>
               <property name="indent">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nama_barang">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>23</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Open Sans</family>
                 <pointsize>15</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #433771;
text-indent: 50px;
color: #dddbe9;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #6656be;</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>14</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Open Sans</family>
                   <pointsize>15</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: #dddbe9</string>
                 </property>
                 <property name="text">
                  <string>Harga Barang:</string>
                 </property>
                 <property name="indent">
                  <number>5</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="minimumSize">
                  <size>
                   <width>25</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Open Sans</family>
                   <pointsize>15</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: #dddbe9</string>
                 </property>
                 <property name="text">
                  <string>Rp.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="harga_barang">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Open Sans</family>
                   <pointsize>15</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: #433771;
text-indent: 50px;
color: #dddbe9;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Open Sans</family>
                   <pointsize>15</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: #dddbe9</string>
                 </property>
                 <property name="text">
                  <string>Jumlah Saldo yang Ingin Dipakai:</string>
                 </property>
                 <property name="indent">
                  <number>5</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="minimumSize">
                  <size>
                   <width>25</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Open Sans</family>
                   <pointsize>15</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: #dddbe9</string>
                 </property>
                 <property name="text">
                  <string>Rp.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="from_savings">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Open Sans</family>
                   <pointsize>15</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: #433771;
text-indent: 50px;
color: #dddbe9;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>300</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Open Sans</family>
                   <pointsize>15</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: #dddbe9</string>
                 </property>
                 <property name="text">
                  <string>Target Menabung Per Bulan:</string>
                 </property>
                 <property name="indent">
                  <number>5</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="minimumSize">
                  <size>
                   <width>25</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Open Sans</family>
                   <pointsize>15</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: #dddbe9</string>
                 </property>
                 <property name="text">
                  <string>Rp.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="target_perbulan">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Open Sans</family>
                   <pointsize>15</pointsize>
                  </font>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: #433771;

color: #dddbe9;</string>
                 </property>
                 <property name="inputMask">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>false</bool>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="calculateButton">
            <property name="font">
             <font>
              <family>Open Sans</family>
              <pointsize>18</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 18pt &quot;Open Sans&quot;;
font-weight: bold;
background-color: #6656be;
color: #dddbe9;</string>
            </property>
            <property name="text">
             <string>Hitung</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="bottomFrame">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="target_hitung">
               <property name="font">
                <font>
                 <family>Open Sans</family>
                 <pointsize>15</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #dddbe9</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="indent">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lower_target">
               <property name="font">
                <font>
                 <family>Open Sans</family>
                 <pointsize>15</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #dddbe9</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="indent">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="upper_target">
               <property name="font">
                <font>
                 <family>Open Sans</family>
                 <pointsize>15</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #dddbe9</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="indent">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="prediction_target">
               <property name="font">
                <font>
                 <family>Open Sans</family>
                 <pointsize>15</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #dddbe9</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="backButton">
              <property name="font">
               <font>
                <family>Open Sans</family>
                <pointsize>18</pointsize>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 18pt &quot;Open Sans&quot;;
font-weight: bold;
background-color: #6656be;
color: #dddbe9;</string>
              </property>
              <property name="text">
               <string>Back</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="saveButton">
              <property name="font">
               <font>
                <family>Open Sans</family>
                <pointsize>18</pointsize>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 18pt &quot;Open Sans&quot;;
font-weight: bold;
background-color: #6656be;
color: #dddbe9;</string>
              </property>
              <property name="text">
               <string>Simpan</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
